src/modules/api/pipeline-task/routes/task.py

```diff
@@ -94,11 +80,10 @@ def update_task():
   # String that identifies this API call, for debugging purposes
   call_id = f'STATUS {get_operation_id_from_name(operation_name)}'
 
-
-  # Update the operation record itself
-  logger.debug(f"[{ call_id }] Updating the pipeline operation record...")
+  # Get a runner object and execution ID representing this job from the operation name
+  logger.debug(f"[{ call_id }] Retrieving the operation...")
   try:
-    op = update_pipeline_operation_record(operation_name)
+    runner, exec_id = get_runner_from_operation_name(operation_name)
```

Comment:
Since the `PipelineOperation` record is now handled implicitly by the `GCPRunner` subclass, we instead parse the operation name into a `Runner` and an execution ID.